NthKey/App/SceneDelegate.swift

```diff
@@ -14,7 +14,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
     private let store = PersistentStore(inMemory: false)
     private lazy var dataManager = DataManager(store: store)
-    private lazy var subsManager = SubscriptionManager()
+    private lazy var subsManager = SubscriptionManager(identifiers: ["com.nthkey.monthly", "com.nthkey.annual"])
```

Comment:
I'd like to start with just monthly. We'll add annual later.

Reply:
fixed in 9b52487b5ce2be3649791267d10e26b385a4aa0a